servers/zms/src/main/java/com/yahoo/athenz/zms/ZMSResources.java

```diff
@@ -17,12 +17,15 @@
     @Path("/domain/{domain}")
     @Produces(MediaType.APPLICATION_JSON)
     public Domain getDomain(@PathParam("domain") String domain) {
+        int code = 200;
+        ResourceContext context = this.delegate.newResourceContext(this.request, this.response);
+        Object timerMetric = this.delegate.startMetricTiming();
```

Comment:
we should probably move this one line up since we should include the time required for context resource creation as well. Another option would be create the object within ResourceContext itself and then just get that on line 45 below.